frontend/src/components/System/AboutLinks.vue

```diff
@@ -1,25 +1,29 @@
 <template>
   <v-list>
-    <v-list-group>
+    <v-item-group>
       <v-list-item
         v-for="linkItem in linkItems"
         :key="linkItem.name"
         rel="noreferrer noopener"
         :href="linkItem.link"
         target="_blank">
-        <v-avatar>
-          <v-icon :icon="linkItem.icon" />
-        </v-avatar>
+        <template v-slot:prepend>
```

Comment:
```suggestion
        <template #prepend>
```

This is the first occurence in the PR, but you used this syntax everywhere else. Revisit those as well please.

Reply:
Done.

My VSCode did not warn me about it.
I am using the development container and installed all suggested extensions.
Something else I should have configured?

Reply:
@Janbong Did you wait for npm install to complete?